browser/components/molecules/ClaimAPrrr/index.js

```diff
@@ -0,0 +1,92 @@
+import React, { Component, PropTypes } from 'react'
+import Link from '../../atoms/Link'
+import Button from '../../atoms/Button'
+import Layout from '../Layout'
+import InspectObject from '../../utils/InspectObject'
+import PendingPrrrs from '../PendingPrrrs'
+import ClaimedPrrrs from '../ClaimedPrrrs'
+import claimPrrr from '../../../actions/claimPrrr'
+import completePrrr from '../../../actions/completePrrr'
+import unclaimPrrr from '../../../actions/unclaimPrrr'
+import GithubUsername from '../../atoms/GithubUsername'
+import Date from '../../atoms/Date'
+import './index.sass'
```

Comment:
please go though this list of imports and remove anything not being used. There are a few in here that we're not using